### PR TITLE
onboarding for node_required accounts

### LINKED ISSUE #3247: Finish setup hint for Alby users
### Feature description

Add a hint for new users (based on internal api data) to finish setting up their account. 

### Describe the solution

We should be able to add another OnboardingCard to the Home page that takes you to the onboarding process of getalby.com

### Describe alternatives

_No response_

### Additional context

_No response_

### Are you working on this?

None

### PR DESCRIPTION
**Pull request by pavanjoshi914**
onboarding screen for accounts which requires node setup

Fixes #3247

### CODE REVIEW

File: src/i18n/locales/en/translation.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
         "review_connection_details": "Please review your connection details.",
         "connection_taking_long": "Trying to connect takes longer than expected... Are your details correct? Is your node reachable?",
         "contact_support": "If you need help please contact support@getalby.com",
+        "node_required": "The extension is configured, but to be able to send and receive payments finish the wallet configuration for your <0>Alby account</0>.",
```
Comment (bumi) [inline src/i18n/locales/en/translation.json:27]:
```suggestion
        "node_required": "The Alby Extension is configured, but to be able to send and receive payments finish the wallet configuration for your <0>Alby Account</0>.",
```

File: src/app/screens/Options/TestConnection/index.tsx [hunk 2]
```diff
@@ -113,9 +117,35 @@ export default function TestConnection() {
                 </div>
 
                 <p className="mt-6 dark:text-gray-400"></p>
-                <p className="mt-6 dark:text-neutral-400">{t("ready")}</p>
```
Comment (bumi) [inline src/app/screens/Options/TestConnection/index.tsx:116]: but now nothing is shown if a node is not required. 

we should keep the success then.

Comment (pavanjoshi914) [reply src/app/screens/Options/TestConnection/index.tsx:116]: hmm. sry! that was an important edge case. fixed

**Comment by pavanjoshi914** [comment]
tAck

**Pull request merged**